src/chainstate/burn/db/sortdb.rs

```diff
@@ -1911,7 +1917,6 @@ impl <'a> SortitionHandleTx <'a> {
         self.put_indexed_begin(&parent_snapshot.sortition_id, &snapshot.sortition_id)?;
 
         let root_hash = self.put_indexed_all(&keys, &values)?;
-        self.indexed_commit()?;
```

Comment:
After this change, where does this commit occur?

Reply:
It happens within the `commit()` method.